backend/src/routes/projects.js

```diff
@@ -14,17 +14,23 @@ const routes = [
     schema: validators.getOne,
     handler: controller.getOne,
   },
+  {
+    method: "PUT",
+    url: `/${what}/:id`,
+    schema: validators.updateOne,
+    handler: controller.updateOne,
+  },
   {
     method: "GET",
     url: `/${what}/:projectId/close-out`,
     schema: validators.getOne,
     handler: controller.getCloseOut,
   },
   {
-    method: "PUT",
-    url: `/${what}/:id`,
-    schema: validators.updateOne,
-    handler: controller.updateOne,
+    method: "POST",
```

Comment:
we should be cautious creating a direct api link to notify, i feel this should only be accessed indirectly.. For now  though i think i'm ok with this, because we will need to answer other questions like. This should only happen once, should be stored in db

Reply:
Yeah I wasn't sure about this either but it didn't seem to fit any of the existing routes.